docs/source/user_guide/12_list_of_environment_variables.rst

```diff
@@ -0,0 +1,184 @@
+List of Env vars
+__________________
+
+
+.. list-table:: User List
+    :widths: 35 100 20
+    :header-rows: 1
+
+    * - Environment Variable Name
+      - Description
+      - Default value
+    * - .. envvar:: RADICAL_BASE
+      - root dir for temporary state files,
+      - `$HOME/.radical/`
+    * - .. envvar:: RADICAL_PILOT_DBURL
+      - MongoDB URL string and mandatory for RP to work.
+      - {NOT_SET}
+    * - .. envvar:: RADICAL_UTILS_NTPHOST
+      - NTP host to be used for profile syncing
+      - `0.pool.ntp.org`
+    * - .. envvar:: RADICAL_PILOT_BULK_CB
+      - Enable bulk callbacks for performance boost. This changes the callback signature.
+      - FALSE
+    * - .. envvar:: RADICAL_PILOT_STRICT_CANCEL
+      - Cancel the tasks immediately. #TODO Didn't understood
+      - {NOT_SET}
+    * - .. envvar:: RADICAL_DEFAULT_LOG_TGT
+      - The log target. #TODO Need more clarification
+      - .
+    * - .. envvar:: RADICAL_DEFAULT_LOG_DIR
+      - The Log dir
+      - $PWD
+    * - .. envvar:: RADICAL_DEFAULT_LOG_LVL
+      - #TODO Description
```

Comment:
#TODO @mtitov I understand these *DEFAULT*, but don't know what to write in their description.

Reply:
Default value of a logging level ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"] to control the debug output.

@andre-merzky do we actually need such env variables as `.._DEFAULT_..`? For example `RADICAL_DEFAULT_LOG_LVL` is treated the same as `RADICAL_LOG_LVL`